docs/confluence-upgrade.md

```diff
@@ -1,8 +1,8 @@
 ## Confluence Rolling Upgrade
-Suppose we have a Confluence cluster with 3 nodes, all running `Confluence xx.xx.0-jdk11`, and an upgrade to 
-`Confluence xx.xx.1-jdk11` is planned. Use these steps to complete the upgrade process: 
+Suppose we have a Confluence cluster with 3 nodes, all running `Confluence xx.xx.0-jdk11`, and an upgrade to
+`Confluence xx.xx.1-jdk11` is planned. Use these steps to complete the upgrade process:
 
-1. Update the `version` and `appVersion` in the Helm chart (`src/main/charts/bitbucket/Chart.yaml`):
+1. Update the `version` and `appVersion` in the Helm chart (`src/main/charts/confluence/Chart.yaml`):
```

Comment:
should we be referencing local directories here? This is actually a general question as I see we reference local paths throughout our readme's. The reason I ask is based on the assumption that we're using the helm remote repo for the charts and not a clone of this repo, see our instructions [here](https://github.com/atlassian-labs/data-center-helm-charts/blob/master/INSTALL.md#installation). With that in mind how would the path(s) change, Im actually not sure 🤔

Reply:
We haven't defined who these docs should target. Should it be the users of the charts or people building their own derivative version of these charts? I assumed the latter since they exist here, in the repo, and a next step talks about calling `helm package`.